Gemfile

```diff
@@ -5,6 +5,9 @@ gem "concurrent-ruby", "~> 1.0.0"
 gem "flay"
 gem "json"
 
+gem "codeclimate-parser-client",
+  path: "/home/app/codeclimate-parser-client"
```

Comment:
this is interesting. were we unable to use the parser-client without requiring here?

Reply:
The require statement you might be looking for is `require "cc/parser"` which is in this diff.